tools/meson-git-contrib.sh

```diff
@@ -4,4 +4,5 @@ git shortlog -s `git describe --abbrev=0`.. | \
         cut -c8- | \
         sed 's/ / /g' | \
         awk '{ print $$0 "," }' | \
+        sed -e 's/ / /g' | \
```

Comment:
I like the non-breaking spaces because then the contribution list is more legible...